client/my-sites/plugins/index.web.js

```diff
@@ -35,7 +36,7 @@ export default function () {
 		'/plugins/browse/:category/:site?',
 		scrollTopIfNoHash,
 		siteSelection,
-		navigation,
+		navigationIfLoggedIn,
```

Comment:
This is needed now since we don't have the sidebar for the logged-out scenarios.

Reply:
Does it make sense to have separate(new) routes for logged in / logged out?

Reply:
Having separate routes sounds good. I'll look into it. Thanks!